### DISCUSSION TITLE
Supress YN0062 platform incompatible warning

### DISCUSSION THREAD

**Discussion by protango** [yarnpkg/berry#2924] [Q&A] [answered]
I'm trying to resolve all the warnings that appear when I run `yarn install` and so far I've resolved them all except:
```
➤ YN0062: │ fsevents@patch:fsevents@npm%3A2.3.2#~builtin<compat/fsevents>::version=2.3.2&hash=1cc4b2 The platform linux is incompatible with this module, link skipped.
```

I know this is a macOS specific module hence why it is not necessary on Linux. But it can't be removed as it is a sub-dependency for some other modules (and also we have macOS developers too).

Is there any way to suppress this warning on Linux?

**Comment by merceyz** [comment, marked as answer]
You can use `logFilters` to suppress log entries https://yarnpkg.com/configuration/yarnrc#logFilters

```yaml
logFilters:
  - code: YN0062
    level: discard

```
